apps/cyberstorm-remix/app/rootRoute/breadcrumbs/getCommunityBreadcrumb.tsx

```diff
@@ -0,0 +1,64 @@
+import { isRecord, NewBreadCrumbsLink } from "@thunderstore/cyberstorm";
```

Comment:
Commenting here as I can't comment on an empty file: Should `breadcrumbs.tsx` contain something or be deleted?